app/services/et_api_handler.rb

```diff
@@ -121,4 +132,8 @@ def self.build_disability_data(full_hash)
       "disability_information": full_hash[:disability_answers][:disability_information]
     }
   end
+
+  def self.parse_validator_response(response)
+    response["status"] == "not_accepted" ? response["errors"].first["code"] : true
```

Comment:
Im not sure this is what you want.  The remote validator can give you multiple errors and the first one might be for a different attribute ?  It might be correct now - but you dont want the situation where you add another validator in the API side and it breaks et3 side

Reply:
Also, you are returning multiple types - either a boolean or whatever "code" is ?  That doesnt seem right.